packages/babel-parser/src/parser/statement.js

```diff
@@ -71,6 +76,33 @@ export default class StatementParser extends ExpressionParser {
     return this.finishNode(node, "InterpreterDirective");
   }
 
+  isLet() {
+    if (!this.isContextual("let")) {
+      return false;
+    }
+    skipWhiteSpace.lastIndex = this.state.pos;
+    const skip = skipWhiteSpace.exec(this.state.input);
+    // $FlowIgnore
+    const next = this.state.pos + skip[0].length;
+    const nextCh = this.state.input.charCodeAt(next);
```

Comment:
We should probably move those lines to a `lookaheadCh` function, since it can be used in many places where we currently use `lookahead()` (and it would be much faster)

Reply:
👍